ansible-scylla-node/tasks/upgrade/upgrade_redhat.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+# Remove existing Scylla repositories
+- name: Remove existing Scylla repositories
+  ansible.builtin.file:
+    path: /etc/yum.repos.d/scylla.repo
+    state: absent    
+  notify: Clean yum metadata  
+  become: true
+  when: upgrade_major
+
+
+# Create Scylla repositories file
+- name: Create Scylla repositories file
+  ansible.builtin.get_url:
+    url: "http://downloads.scylladb.com/rpm/centos/scylla-{{ scylla_upgrade['major_version'] }}.repo"
```

Comment:
this url might also change, should be taken from some global variable

Reply:
The repo URL is dynamically generated based on the selected version. The global variable is fixed string.